tests/tests/src/preserve_jsx_test.mjs

```diff
@@ -50,43 +60,63 @@ let baseProps = {
 
 let newrecord = {...baseProps};
 
-let _unary_element_with_spread_props = <input {...newrecord}  type={"text"}/>;
+let _unary_element_with_spread_props = <input {...newrecord} 
+  type={"text"}
+/>;
 
 let newrecord$1 = {...baseProps};
 
-let _container_with_spread_props = <div {...newrecord$1}  title={"barry"} className={"barry"}>
-{"Hello, world!"}
-<input type={"text"}/>
+let _container_with_spread_props = <div {...newrecord$1} 
+  title={"barry"}
+  className={"barry"}
+>
+  {"Hello, world!"}
+  <input
```

Comment:
As a further improvement, maybe we could break only if there is more than one prop?